sound/soc/sof/topology.c

```diff
@@ -300,11 +300,15 @@ static int sof_control_load_bytes(struct snd_soc_component *scomp,
 	struct sof_ipc_ctrl_data *cdata;
 	struct snd_soc_tplg_bytes_control *control =
 		(struct snd_soc_tplg_bytes_control *)hdr;
+	/*
 	const int max_size = SOF_IPC_MSG_MAX_SIZE -
 		sizeof(const struct sof_ipc_ctrl_data);
+	*/
+
+	const int max_size = SOF_IPC_MSG_MAX_SIZE * 10;
 
 	/* init the get/put bytes data */
-	scontrol->size = SOF_IPC_MSG_MAX_SIZE;
+	scontrol->size = max_size;
```

Comment:
we probably need to change the logic so that we only allocate what is strictly necessary, not the max size.

Reply:
yes, sorry that there are so much temp hacks, not my code originally, so I was not sure how to touch/change before discussing

Reply:
@plbossart should be now so that we only allocate needed value